models/issue.go

```diff
@@ -1062,15 +1062,15 @@ func UpdateIssueMentions(issueID int64, mentions []string) error {
 // IssueStats represents issue statistic information.
 type IssueStats struct {
 	OpenCount, ClosedCount int64
-	AllCount               int64
+	YourRepositoriesCount  int64
```

Comment:
As a side note, personally I'd think the most interesting views are "Created by you" "Assigned to you" and "Mentions you" (in other words, no "Your repositories" view and an added "Mentions you" view). This is perhaps a discussion for another time, for this PR I left this functionality unchanged.